Packs/VirusTotal/Integrations/VirusTotalV3/VirusTotalV3.py

```diff
@@ -96,6 +96,176 @@
 }
 
 
+class VTFile(Common.File):
+    """VT File Indicator."""
+
+    def __init__(
+            self,
+            dbot_score,
+            count_vt_vendors_which_flagged_malicious=None,
+            vt_vendors_which_flagged_malicious=None,
+            vt_detection_names=None,
+            **kwargs
+    ):
+        super().__init__(
+            dbot_score,
+            **kwargs
+        )
+
+        self.count_vt_vendors_which_flagged_malicious = count_vt_vendors_which_flagged_malicious
+        self.vt_vendors_which_flagged_malicious = vt_vendors_which_flagged_malicious
+        self.vt_detection_names = vt_detection_names
+
+    def to_context(self):
+        context = super().to_context()
+        file_context = context[super().CONTEXT_PATH]
+
+        file_context['VTVendors'] = {}
+
+        if self.count_vt_vendors_which_flagged_malicious is not None:
+            file_context['VTVendors']['EngineDetections'] = self.count_vt_vendors_which_flagged_malicious
+
+        if self.vt_vendors_which_flagged_malicious is not None:
+            file_context['VTVendors']['EngineVendors'] = self.vt_vendors_which_flagged_malicious
+
+        if self.vt_detection_names is not None:
+            file_context['VTVendors']['EngineDetectionNames'] = self.vt_detection_names
+
+        if not file_context['VTVendors']:
+            file_context.pop('VTVendors', None)
+
+        return context
+
+
+class VTIP(Common.IP):
+    """VT IP Indicator."""
+
+    def __init__(
+            self,
+            ip,
+            dbot_score,
+            count_vt_vendors_which_flagged_malicious=None,
+            vt_vendors_which_flagged_malicious=None,
+            vt_detection_names=None,
+            **kwargs
+    ):
+        super().__init__(
+            ip,
+            dbot_score,
+            **kwargs
+        )
+
+        self.count_vt_vendors_which_flagged_malicious = count_vt_vendors_which_flagged_malicious
+        self.vt_vendors_which_flagged_malicious = vt_vendors_which_flagged_malicious
+        self.vt_detection_names = vt_detection_names
+
+    def to_context(self):
+        context = super().to_context()
+        file_context = context[super().CONTEXT_PATH]
+
+        file_context['VTVendors'] = {}
+
+        if self.count_vt_vendors_which_flagged_malicious is not None:
+            file_context['VTVendors']['EngineDetections'] = self.count_vt_vendors_which_flagged_malicious
+
+        if self.vt_vendors_which_flagged_malicious is not None:
+            file_context['VTVendors']['EngineVendors'] = self.vt_vendors_which_flagged_malicious
+
+        if self.vt_detection_names is not None:
+            file_context['VTVendors']['EngineDetectionNames'] = self.vt_detection_names
+
+        if not file_context['VTVendors']:
```

Comment:
Consider using the same convention as in other if statements
`if not file_context['VTVendors']:`
or
`if file_context['VTVendors'] is not None:`